angular-todo/e2e/embedded-login.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { test, expect } from '@playwright/test';
+
+test('embedded login angular', async ({ page }) => {
+  await page.goto('https://localhost:8443/', { waitUntil: 'networkidle' });
+  await page.getByRole('link', { name: 'Sign In', exact: true }).click();
+
+  //failed login
+  await page.getByLabel('User Name').fill('notauser');
+  await page.getByLabel('Password').fill('notapassword');
+
+  await page.getByPlaceholder('Password').press('Enter');
+  await expect(page.getByText('Login failure')).toBeVisible();
+
+  //successful login
+  await page.goto('https://localhost:8443/home', { waitUntil: 'networkidle' });
```

Comment:
I think waitUntil network idle is discouraged. https://playwright.dev/docs/api/class-frame#frame-goto We should find an alternative.

Reply:
set to default and worked(load)